ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -251,7 +253,7 @@ extern "C" {
                                           classList: *mut *mut *mut nsIAtom)
      -> u32;
     pub fn Gecko_GetServoDeclarationBlock(element: *mut RawGeckoElement)
-     -> *mut ServoDeclarationBlock;
+     -> ServoDeclarationBlockBorrowed;
```

Comment:
Leave it as-is for now. If this gets fixed, it will be months from now (since we're on stable), and by then I expect to have better ownership semantics for the bindings in.